[2017-03-09 05:57:25,031: WARNING/MainProcess] consumer: Connection to broker lost. Trying to re-establish the connection...
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/celery/worker/consumer/consumer.py", line 318, in start
    blueprint.start(self)
  File "/usr/lib/python2.7/site-packages/celery/bootsteps.py", line 119, in start
    step.start(parent)
  File "/usr/lib/python2.7/site-packages/celery/worker/consumer/consumer.py", line 594, in start
    c.loop(*c.loop_args())
  File "/usr/lib/python2.7/site-packages/celery/worker/loops.py", line 88, in asynloop
    next(loop)
  File "/usr/lib/python2.7/site-packages/kombu/async/hub.py", line 345, in create_loop
    cb(*cbargs)
  File "/usr/lib/python2.7/site-packages/kombu/transport/base.py", line 236, in on_readable
    reader(loop)
  File "/usr/lib/python2.7/site-packages/kombu/transport/base.py", line 218, in _read
    drain_events(timeout=0)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 464, in drain_events
    return self.blocking_read(timeout)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 469, in blocking_read
    return self.on_inbound_frame(frame)
  File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 68, in on_frame
    callback(channel, method_sig, buf, None)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 473, in on_inbound_method
    method_sig, payload, content,
  File "/usr/lib/python2.7/site-packages/amqp/abstract_channel.py", line 142, in dispatch_method
    listener(*args)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 593, in _on_close
    self._x_close_ok()
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 608, in _x_close_ok
    self.send_method(spec.Connection.CloseOk, callback=self._on_close_ok)
  File "/usr/lib/python2.7/site-packages/amqp/abstract_channel.py", line 64, in send_method
    conn.frame_writer(1, self.channel_id, sig, args, content)
  File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 174, in write_frame
    write(view[:offset])
  File "/usr/lib/python2.7/site-packages/amqp/transport.py", line 269, in write
    self._write(s)
  File "/usr/lib64/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 104] Connection reset by peer
[2017-03-09 05:57:25,366: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111] Connection refused.
Trying again in 2.00 seconds...

[2017-03-09 05:58:08,897: INFO/MainProcess] Connected to amqp://guest:**@127.0.0.1:5672//
[2017-03-09 05:58:08,912: INFO/MainProcess] mingle: searching for neighbors
[2017-03-09 05:58:09,937: INFO/MainProcess] mingle: all alone
[2017-03-09 05:58:09,956: WARNING/MainProcess] /usr/lib/python2.7/site-packages/celery/fixups/django.py:202: UserWarning: Using settings.DEBUG leads to a memory leak, never use this setting in production environments!
  warnings.warn('Using settings.DEBUG leads to a memory leak, never '
[2017-03-09 05:58:09,956: INFO/MainProcess] worker1@goldenspeaker.las.iastate.edu ready.
[2017-03-09 05:58:50,914: INFO/MainProcess] Received task: speech.tasks.build_sabr_model[46a8ab0c-4fac-493c-8e98-b024061d48ac]  
[2017-03-09 06:07:20,287: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[e75e0867-8141-4bd3-b00e-38f109140609]  
[2017-03-09 06:13:49,787: WARNING/MainProcess] consumer: Connection to broker lost. Trying to re-establish the connection...
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/celery/worker/consumer/consumer.py", line 318, in start
    blueprint.start(self)
  File "/usr/lib/python2.7/site-packages/celery/bootsteps.py", line 119, in start
    step.start(parent)
  File "/usr/lib/python2.7/site-packages/celery/worker/consumer/consumer.py", line 594, in start
    c.loop(*c.loop_args())
  File "/usr/lib/python2.7/site-packages/celery/worker/loops.py", line 88, in asynloop
    next(loop)
  File "/usr/lib/python2.7/site-packages/kombu/async/hub.py", line 345, in create_loop
    cb(*cbargs)
  File "/usr/lib/python2.7/site-packages/kombu/transport/base.py", line 236, in on_readable
    reader(loop)
  File "/usr/lib/python2.7/site-packages/kombu/transport/base.py", line 218, in _read
    drain_events(timeout=0)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 464, in drain_events
    return self.blocking_read(timeout)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 469, in blocking_read
    return self.on_inbound_frame(frame)
  File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 68, in on_frame
    callback(channel, method_sig, buf, None)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 473, in on_inbound_method
    method_sig, payload, content,
  File "/usr/lib/python2.7/site-packages/amqp/abstract_channel.py", line 142, in dispatch_method
    listener(*args)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 593, in _on_close
    self._x_close_ok()
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 608, in _x_close_ok
    self.send_method(spec.Connection.CloseOk, callback=self._on_close_ok)
  File "/usr/lib/python2.7/site-packages/amqp/abstract_channel.py", line 64, in send_method
    conn.frame_writer(1, self.channel_id, sig, args, content)
  File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 174, in write_frame
    write(view[:offset])
  File "/usr/lib/python2.7/site-packages/amqp/transport.py", line 269, in write
    self._write(s)
  File "/usr/lib64/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 104] Connection reset by peer
[2017-03-09 06:13:49,846: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111] Connection refused.
Trying again in 2.00 seconds...

[2017-03-09 06:14:16,655: INFO/MainProcess] Connected to amqp://guest:**@127.0.0.1:5672//
[2017-03-09 06:14:16,681: INFO/MainProcess] mingle: searching for neighbors
[2017-03-09 06:14:17,734: INFO/MainProcess] mingle: all alone
[2017-03-09 06:14:17,775: WARNING/MainProcess] /usr/lib/python2.7/site-packages/celery/fixups/django.py:202: UserWarning: Using settings.DEBUG leads to a memory leak, never use this setting in production environments!
  warnings.warn('Using settings.DEBUG leads to a memory leak, never '
[2017-03-09 06:14:17,775: INFO/MainProcess] worker1@goldenspeaker.las.iastate.edu ready.
[2017-03-09 06:14:56,591: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[b8f583ec-fd38-49e7-b665-ea349369434c]  
[2017-03-09 06:17:59,200: WARNING/MainProcess] consumer: Connection to broker lost. Trying to re-establish the connection...
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/celery/worker/consumer/consumer.py", line 318, in start
    blueprint.start(self)
  File "/usr/lib/python2.7/site-packages/celery/bootsteps.py", line 119, in start
    step.start(parent)
  File "/usr/lib/python2.7/site-packages/celery/worker/consumer/consumer.py", line 594, in start
    c.loop(*c.loop_args())
  File "/usr/lib/python2.7/site-packages/celery/worker/loops.py", line 88, in asynloop
    next(loop)
  File "/usr/lib/python2.7/site-packages/kombu/async/hub.py", line 345, in create_loop
    cb(*cbargs)
  File "/usr/lib/python2.7/site-packages/kombu/transport/base.py", line 236, in on_readable
    reader(loop)
  File "/usr/lib/python2.7/site-packages/kombu/transport/base.py", line 218, in _read
    drain_events(timeout=0)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 464, in drain_events
    return self.blocking_read(timeout)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 469, in blocking_read
    return self.on_inbound_frame(frame)
  File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 68, in on_frame
    callback(channel, method_sig, buf, None)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 473, in on_inbound_method
    method_sig, payload, content,
  File "/usr/lib/python2.7/site-packages/amqp/abstract_channel.py", line 142, in dispatch_method
    listener(*args)
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 593, in _on_close
    self._x_close_ok()
  File "/usr/lib/python2.7/site-packages/amqp/connection.py", line 608, in _x_close_ok
    self.send_method(spec.Connection.CloseOk, callback=self._on_close_ok)
  File "/usr/lib/python2.7/site-packages/amqp/abstract_channel.py", line 64, in send_method
    conn.frame_writer(1, self.channel_id, sig, args, content)
  File "/usr/lib/python2.7/site-packages/amqp/method_framing.py", line 174, in write_frame
    write(view[:offset])
  File "/usr/lib/python2.7/site-packages/amqp/transport.py", line 269, in write
    self._write(s)
  File "/usr/lib64/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 104] Connection reset by peer
[2017-03-09 06:17:59,233: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111] Connection refused.
Trying again in 2.00 seconds...

[2017-03-09 06:18:26,009: INFO/MainProcess] Connected to amqp://guest:**@127.0.0.1:5672//
[2017-03-09 06:18:26,024: INFO/MainProcess] mingle: searching for neighbors
[2017-03-09 06:18:27,055: INFO/MainProcess] mingle: all alone
[2017-03-09 06:18:27,090: WARNING/MainProcess] /usr/lib/python2.7/site-packages/celery/fixups/django.py:202: UserWarning: Using settings.DEBUG leads to a memory leak, never use this setting in production environments!
  warnings.warn('Using settings.DEBUG leads to a memory leak, never '
[2017-03-09 06:18:27,090: INFO/MainProcess] worker1@goldenspeaker.las.iastate.edu ready.
[2017-03-09 06:18:44,872: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[e7915944-9dc4-43c4-9a19-2418e629914c]  
[2017-03-09 06:20:19,618: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[ab9f9fa6-d8b2-4737-a38e-0bf9e965a3df]  
[2017-03-09 06:25:26,705: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[9632096c-ed9e-4d45-8b6f-65854bbf629a]  
[2017-03-09 13:40:13,801: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[9fad6efd-cde5-4b92-a23c-ac4396326908]  
[2017-03-09 15:35:06,369: INFO/MainProcess] Received task: speech.tasks.build_sabr_model[26752894-e6de-44c2-896f-2d502d0bd755]  
[2017-03-09 15:36:33,324: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[50ab48d3-bdc3-4178-8cdf-2ab70e5e8ed4]  
[2017-03-09 16:47:26,235: INFO/MainProcess] Received task: speech.tasks.build_sabr_model[74ac6695-e9d0-49ca-880a-83a75a3c2316]  
[2017-03-09 16:48:57,468: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[849aafad-50d1-45e7-9208-10c4a3caaf83]  
[2017-03-09 16:58:23,445: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[af795aa7-e46c-48da-b262-47d0971fcb6a]  
[2017-03-09 17:06:45,276: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[7c19eb0b-32a4-45f0-a953-37a1a5a3e8d0]  
[2017-03-09 21:59:02,760: INFO/MainProcess] Received task: speech.tasks.build_sabr_model[472414e7-e251-4532-ab59-0797e6bec8d0]  
[2017-03-09 22:02:46,553: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[badfee13-7c13-40b0-9fa7-39f8c3732f10]  
[2017-03-09 22:06:50,868: INFO/MainProcess] Received task: speech.tasks.synthesize_sabr[cfce578f-281a-4d79-84f8-87d3f05741d8]  
